packages-web/maps/tests/e2e/specs/google.spec.ts

```diff
@@ -18,7 +18,7 @@ describe("Google Maps", () => {
 
             const markers = mapPage.googleMarkers;
 
-            expect(markers.length).toBe(3);
+            browser.waitUntil(() => markers.length === 3, 5000);
```

Comment:
that is meaningless, since you get markers array outside of waitUntil

Reply:
Fixed